docs/api-workspace/npmscripts/affected-e2e.md

```diff
@@ -36,26 +36,12 @@ Latest commit of the current branch (usually HEAD)
 
 Show help
 
-### maxParallel
```

Comment:
why is this removed? all affected commands should have this option?

Reply:
The parallel options aren't added for the [affected:e2e](https://github.com/nrwl/nx/blob/master/packages/workspace/src/command-line/nx-commands.ts#L71) command. I assumed there was reason, but I'm not sure why myself.

Reply:
I think this is a mistake. Would you like me to add the parallel options?

Reply:
Yup. All the affected command support the parallel option, so it should be reflected in the docs.

Reply:
There's a port in use error when running `affected:e2e` in parallel.

![Screen Shot 2019-07-17 at 11 03 03 PM](https://user-images.githubusercontent.com/14145352/61428295-21d48b80-a8e7-11e9-8f1a-9dd4a77d6354.png)

### Steps to Reproduce

1. Generate a workspace with a single Angular app `npm init nx-workspace myworkspace`
2. Generate a second Angular app `ng g app my-app`
3. `npm run affected:e2e -- --all --headless --parallel`

Ideally the builder would handle this by setting port to 0, but I'm having trouble determining when to do that. The obvious condition is when `parallel` is true, but that option is always false in the builder regardless of whether you specify it as true on the command line because it's [filtered out](https://github.com/nrwl/nx/blob/master/packages/workspace/src/command-line/affected.ts#L55) before being passed to the builder.

Would it be alright if this is handled as a separate issue and this PR can be merged as is?